src/App.js

```diff
@@ -34,7 +34,8 @@ class App extends React.Component {
 
     if (response.list) {
       this.setState({
-        weatherData: response
+        weatherData: response,
+        data: response
```

Comment:
Do you need to save the same thing under 2 different names? It's just repeating data unnecessarily. You can save it all under `weatherData` and use that throughout.